classes/PublishPress/Permissions/UI/SettingsTabCore.php

```
@@ -239,15 +239,21 @@ public function optionsUI()
 
                                             $url = Settings::pluginInfoURL('capability-manager-enhanced');
 
-                                            $caption = $define_create_posts_cap
-                                                ? 'Post creation capabilities will also be enforced for all Filtered Post Types. To adjust this, install %3$sPublishPress Capabilities%4$s.'
-                                                : 'To enforce capability requirements for post creation, install %3$sPublishPress Capabilities%4$s.';
+                                            $caption = ($define_create_posts_cap)
+                                                ? __(
+                                                    'Post creation capabilities will also be enforced for all Filtered Post Types. To adjust this, install %3$sPublishPress Capabilities%4$s.',
+                                                    'press-permit-core'
+                                                )
+                                                : __(
+                                                    'To enforce capability requirements for post creation, install %3$sPublishPress Capabilities%4$s.',
+                                                    'press-permit-core'
+                                                );
```

In this previous code i got error on running phpcs. I think we need add line to ignore error from phpcs.

---

I think this looks good